src/qbittorrent.exe.manifest

```diff
@@ -36,4 +36,10 @@
           <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
       </application>
   </compatibility>
+  <!-- Enable long paths that exceed MAX_PATH in length -->
+  <asmv3:application>
+    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">
+      <longPathAware>true</longPathAware>
```

Comment:
Maybe we should follow word-for-word: `<ws2:longPathAware>true</ws2:longPathAware>`
https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests#longpathaware

Reply:
The CI builds would fails saying invalid manifest. Tried that already.
Also local builds failed as well on my end.

Reply:
OK, all that left is to test this PR and see if it really works before we merge it...

Reply:
I think the relevant test case are:
- Downloading fully and rechecking
- Downloading fully, restarting and rechecking
- Repeating the 2 cases above but with the `.!qB` extension option enabled

Reply:
All 3 worked.